uint8 channel
float32 angle_deg
---
bool success
string message
